feature-libs/organization/administration/components/unit/links/approvers/unit-approver-list.service.ts

```diff
@@ -22,7 +26,8 @@ export class UnitApproverListService extends OrganizationSubListService<
 
   constructor(
     protected tableService: TableService,
-    protected unitService: OrgUnitService
+    protected unitService: OrgUnitService,
+    protected userService: B2BUserService
```

Comment:
the name should be probably `b2bUserService: B2BUserService`, since we have `userService` already and it can be confusing

Reply:
Both are in usage, as we have separated libs, we can consider avoid of b2b suffixes where they are not necessary

![image](https://user-images.githubusercontent.com/8040837/96995951-51813300-152f-11eb-9294-e19e41f8823f.png)